personal/privacyidea/class_mfaAccount.inc

```diff
@@ -192,8 +192,8 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
-        if (!$this->is_account && $this->acl_is_writeable("")) {
-            $this->is_account = true;
+        if (!isset($this->parent)) {
+            $this->uid = $this->ui->uid;
```

Comment:
Not needed, we use GetUid() everywhere, so having $this->uid only in execute does not make sense.

Reply:
Actually, we do use it here: https://github.com/gosa-project/gosa-plugins-privacyidea/blob/d05d41b66eee1bde62267779d499eedc37d49141/personal/privacyidea/class_mfaAccount.inc#L834
although that could be replaced. Is there no possibility it is used outside the plugin (after `execute()`), since it is a property of `class plugin`?